app/controlers/pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class PagesController < ApplicationController
```

Comment:
Looks like maybe this file was an accidental copy/paste?  This "app/controlers/pages_controller.rb" file ("controlers" with one L) duplicates the existing "app/controllers/pages_controller.rb" file ("controllers" with two Ls).

Reply:
File removed.